gammapy/utils/pbar.py

```diff
@@ -0,0 +1,25 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Utilities for progress bar display"""
+from contextlib import contextmanager
+
+try:
+    from tqdm import tqdm
+except ImportError:
+    class tqdm():
+        def __init__(
+                self,
+                total=0,
+                mininterval=0,
+                disable=True,
+                desc=""
+        ):
+            pass
+        def update(self, x):
+            pass
+
+@contextmanager
+def pbar(total=None, show_pbar=True, desc=None):
+    if total == None and show_pbar == True:
```

Comment:
replace `if total == None` by `if total is None` since in Python `None` is not a value but an object.